modules/ROOT/pages/6.7.2-release-notes.adoc

```diff
@@ -37,7 +37,13 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 === Applying heading formatting to the content of the `summary` element extended its application to the content of the parent `details` element.
 // TINY-10154
 
-// CCFR
+In previous versions of {productname}, the accordion body wrapper was not functioning as expected.
```

Comment:
I guess this should be summary since that is what now supports H1-H6 elements.

```suggestion
In previous versions of {productname}, the accordion summary wrapper was not functioning as expected.
```

Reply:
Should it be just `accordion summary` instead of `accordion summary wrapper`?